### PR TITLE
#6 add authentication with fb and some tests

### PR DESCRIPTION
**Pull request by Mywe4ka**

### CODE REVIEW

File: app/assets/javascripts/facebook.js.erb [hunk 1]
```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2013 Google Inc.
```
Comment (VeronikaSalo) [inline app/assets/javascripts/facebook.js.erb]: Please remove this file as far it is not used anywhere.

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class User < ActiveRecord::Base
+  attr_accessible :name, :oauth_expires_at, :oauth_token, :provider, :uid
+
+	def self.from_omniauth(auth)
```
Comment (VeronikaSalo) [inline app/models/user.rb:4]: Please change to correct indentation.

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -1,14 +1,20 @@
...
+  <title>Soundcloud</title>
   <%= stylesheet_link_tag    "application", :media => "all" %>
   <%= javascript_include_tag "application" %>
   <%= csrf_meta_tags %>
 </head>
 <body>
-
-<%= yield %>
-
+	<div id="user_nav">
```
Comment (VeronikaSalo) [inline app/views/layouts/application.html.erb:10]: Fix the indentation here as well.

File: log/test.log [hunk 4]
```diff
@@ -0,0 +1,1020 @@
+Connecting to database specified by database.yml
```
Comment (VeronikaSalo) [inline log/test.log]: Add /log files to .gitignore. db/test.sqlite3 as well. We do not need to commit them.

File: spec/models/user_spec.rb [hunk 5]
```diff
@@ -0,0 +1,6 @@
+require 'spec_helper'
+
+describe User do
+
```
Comment (VeronikaSalo) [inline spec/models/user_spec.rb]: Where the tests itself??? When you leave place for future rework please add #ToDo comment

File: test/unit/user_test.rb [hunk 6]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  # test "the truth" do
```
Comment (VeronikaSalo) [inline test/unit/user_test.rb:4]: @Mywe4ka  do we need this file here?

File: tmp/pids/server.pid [hunk 7]
```diff
@@ -0,0 +1 @@
+1510
```
Comment (VeronikaSalo) [inline tmp/pids/server.pid]: @Mywe4ka Please add all /tmp files to .gitignore.
And for the future go to /files submenu on your PR and check all items you've added.

**Comment by VeronikaSalo** [comment]
@Mywe4ka Please look at all concerns above and rebase current PR. It would be great you to squash all your commits into one.

File: test/unit/user_test.rb (hunk 6 above)
Comment (Mywe4ka) [inline test/unit/user_test.rb:4]: @VeronikaSalo let's keep it for now. M/b it will be helpful for future.

**Comment by Mywe4ka** [comment]
@VeronikaSalo 
I've resolved conflicts. You can check.

**Pull request merged**